disk-deactivate/disk-deactivate.jq

```diff
@@ -26,6 +26,9 @@ def remove:
 def deactivate:
   if .type == "disk" then
     [
+      # If this disk is a member of raid, stop that raid
+      "mdadm --stop $(lsblk \(.path) -l -p -o type,name | awk 'match($1,\"raid.*\") {print $2}')",
```

Comment:
This should print no error in case there is no device:

```suggestion
      "lsblk /dev/nvme0n1 -l -p -o type,name | awk 'match($1,"raid.*") {print $2}' | xargs -r mdadm --stop",
```

Reply:
What if the raid is still mounted? Can it be stopped?

Reply:
Also this might be less of a concern for the common installer use case where mdadm's are assembled but not mounted.

Reply:
Ah. We could also do an umount of the raid device prior to stopping? `umount /dev/md1` should work.

Reply:
Do we want to do that? We don't umount normal filesystems here either? 

I mainly thought about the scenario where one uses disko with nixos-anywhere: after kexec, the raid is "running", but not mounted.

Reply:
We do unmount other filesystems in the same jq script that are used by the block device.

Reply:
Ah, you are right. Thanks!